src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -21,6 +21,13 @@
 #include "core/common/device.h"
 #include <boost/optional/optional.hpp>
 #include <boost/property_tree/json_parser.hpp>
+#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/split.hpp>
+#include <boost/algorithm/string/join.hpp>
+#include <boost/lexical_cast.hpp>

```

Comment:
No, please do not use boost::lexical cast

Reply:
Removed.